ceph-docker-pull-requests/build/build

```diff
@@ -17,7 +17,12 @@ sudo apt-get install -y --force-yes xfsprogs
 sudo "$WORKSPACE"/ceph-docker/travis-builds/purge_cluster.sh
 # XXX purge_cluster only stops containers, it doesn't really remove them so try to
 # remove them for real
-sudo docker rm -f $@ $(sudo docker ps -a -q)
+container_count=$(sudo docker ps -a -q | wc -l)
```

Comment:
this could just be:

    containers_to_remove=$(sudo docker ps -a -q)

So that you can reuse it later in both line 22 and 23